sormas-api/pom.xml

```diff
@@ -30,8 +30,8 @@
 	</build>
 
 	<properties>
-		<maven.compiler.source>1.7</maven.compiler.source>
-		<maven.compiler.target>1.7</maven.compiler.target>
+		<maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
The compiler source for the API has to be 1.7 since this is the latest version which the sormas-app supports.